scripts/globals/mobskills/Bombilation.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------
+--  Bombilation
+--
+--  Description: Resets TP of all targets in an area of effect.
+--  Type: Enfeebling
+--  Ignores Shadows
+--  Range: Unknown radial
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+   local reset = 0;
```

Comment:
yer indent is off again. And your blink line is on the wrong side of your dashed line above, just fyi. Hope I don't sound nitpicky, its mainly a cleanup pr for formatting and since I am ocd about this stuff I am especially interested.